main.go

```diff
@@ -108,26 +212,60 @@ func main() {
 	}
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
-
-	fileData, err := ioutil.ReadFile(filePath)
+	
+	var subsetPartition map[int][]int
+	err, test := readTestFile(filePath)
 	if err != nil {
 		fatal(err)
 	}
 
-	var test Test
-	var summary Summary
+	rand.Seed(time.Now().UTC().UnixNano())
+	err2, subsetPartition := partition(test.Config)
```

Comment:
One more small thing -- reason for using `err`, then `err2`, then `err` again? Not sure what the best practice is here, I think I've seen `err` reused usually though.